haunt-roll-fail/arcs/game-lore.scala

```diff
@@ -83,6 +87,59 @@ object LoreExpansion extends Expansion {
                 .some(systems)(s => f.at(s).cities./(c => TaxAction(f, x, s, c, true, then).as(c, "in", s, |(board.resource(s)).%(game.available)./(r => ("for", r, Image(r.name, styles.token))))(g).!(f.taxed.has(c), "taxed")))
                 .cancel
 
+        case PruneMainAction(f, x, then) =>
+            val prefix = f.short + "-"
+            def suffix(s : System) = f.rivals.exists(_.rules(s)).??("-damaged")
+
+            Ask(f)
+                .group("Prune".hl)
+                .some(systems)(s => f.pool(Starport).??(f.at(s).cities)./(u => PruneCityAction(f, x, s, u, then).as(City.of(f), Image(prefix + "city" + f.damaged.has(u).??("-damaged"), styles.qbuilding), "in", s)))
+                .some(systems)(s => f.at(s).starports./(u => PruneStarportAction(f, x, s, u, then).as(Starport.of(f), Image(prefix + "starport" + f.damaged.has(u).??("-damaged"), styles.qbuilding), "in", s)))
+                .cancel
+
+        case PruneCityAction(f, x, s, u, then) =>
+            f.pay(x)
+
+            val u2 = f.reserve --> Starport.of(f)
+
+            if (f.damaged.has(u))
+                f.damaged :+= u2
```

Comment:
Probably need to remove `u` too from `damaged`, so it wouldn't be build damaged somewhere else.

Reply:
Fixed.